test/e2e/webdriver/by.js

```diff
@@ -0,0 +1,24 @@
+const { By: SeleniumBy } = require('selenium-webdriver');
+const { parse, xPathBuilder } = require('css-to-xpath');
+
+module.exports = class By {
+  constructor(selector) {
+    this.selector = selector;
+  }
+
+  text(text) {
+    return new By(this.selector.where(xPathBuilder.text().equals(text)));
+  }
+
+  static tag(tag) {
+    return new By(parse(tag));
+  }
+
+  static css(selector) {
+    return new By(parse(selector));
```

Comment:
I don't know how this works on Playwright, but on Selenium I've been trying to avoid using xpath because I've heard it was fairly slow in comparison to ID or CSS selectors. Normalizing to xpath might make things slower? Not sure :thinking:

Reply:
til